src/TailwindBinary.php

```diff
@@ -89,6 +89,17 @@ private function downloadExecutable(): void
         chmod($targetPath, 0777);
     }
 
+    private function getLatestVersion(): string
+    {
+        try {
+            $response = $this->httpClient->request('GET', 'https://api.github.com/repos/tailwindlabs/tailwindcss/releases/latest');
+
+            return $response->toArray()['name'] ?? self::VERSION;
+        } catch (\Throwable) {
+            return self::VERSION;
```

Comment:
How about rename this constant to `DEFAULT_VERSION` (or `FALLBACK_VERSION`) for clarity?

Though, we want to allow users to specify their own version via config, and probably this fallback option would be even redundant then IMO. Thoughts? :)

Reply:
I was too hesitating how to name it. I decided to stick with the original name, hoping that someone can come up with a better alternative.

Reply:
@bocharsky-bw `DEFAULT_VERSION` sounds more acceptable for me. WDYT?

Reply:
Yeah, agree on `DEFAULT_VERSION`... at least let's start with it, it's a private const anyway :)